public_html/wp-content/mu-plugins/camptix-tweaks/camptix-tweaks.php

```diff
@@ -675,7 +675,7 @@ function switch_email_template( $template_slug ) {
  * @return string
  */
 function get_global_sponsors_string() {
-	switch_to_blog( BLOG_ID_CURRENT_SITE ); // central.wordpress.org
+	switch_to_blog( WORDCAMP_ROOT_BLOG_ID ); // central.wordcamp.org.
```

Comment:
Other than this line are all linter auto fix.

Reply:
Can you recreate the PR without the format changes?

Reply:
done